cdap-data-fabric/src/test/java/co/cask/cdap/data/hbase/HBaseTestBase.java

```diff
@@ -55,7 +55,16 @@ public String getZkConnectionString() {
 
   // Test startup / teardown
 
-  public abstract void startHBase() throws Exception;
+  public final void startHBase() throws Exception {
+    // Tune down the connection thread pool size
+    getConfiguration().setInt("hbase.hconnection.threads.core", 5);
+    getConfiguration().setInt("hbase.hconnection.threads.max", 10);

```

Comment:
Acutally, that's not complete true.  From the thead dumps, it looks like the HConnection pool is getting allocated somewhere.  Looking in to why that is.